projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -162,34 +163,28 @@ export class IgxDateRangePickerComponent extends PickerBaseDirective
     @Input()
     public override overlaySettings: OverlaySettings;
 
-    /**
-     * The format used when editable inputs are not focused.
-     *
-     * @remarks
-     * Uses Angular's DatePipe.
-     *
-     * @example
-     * ```html
-     * <igx-date-range-picker displayFormat="EE/M/yy"></igx-date-range-picker>
-     * ```
-     *
-     */
-    @Input()
-    public override displayFormat: string;
 
     /**
      * The expected user input format and placeholder.
      *
-     * @remarks
-     * Default is `"'MM/dd/yyyy'"`
-     *
      * @example
      * ```html
      * <igx-date-range-picker inputFormat="dd/MM/yy"></igx-date-range-picker>
      * ```
      */
     @Input()
-    public override inputFormat: string;
+    public override set inputFormat(value: string) {
+        if (value) {
+            this._inputFormat = value;
+            if (this.hasProjectedInputs) {
+                this.updateInputFormat();
+            }
+        }
```

Comment:
Same logic with the picker base - the old functionality around `updateInputFormat` didn't care if there was a value or not.
Also you could use `super.inputFormat` to avoid rewriting the same get/set logic, IIRC it should work with our compile options. It's already in the state directive.

Reply:
Same could go for the `displayFormat`, assuming the override is for documentation snippet purposes?